src/views/ActivityAppFeed.vue

```diff
@@ -208,12 +208,10 @@ watch(props, () => {
 	&__container {
 		display: flex;
 		flex-direction: column;
-
 		height: 100%;
-		width: min(100%, 924px);
-		max-width: 924px;
-		margin: 0 auto;
-		padding-inline: 12px;
+		// Align with app navigation toggle
+		margin: var(--app-navigation-padding, 8px) 0 0 calc(2 * var(--app-navigation-padding, 8px) + 44px);
```

Comment:
```suggestion
		margin: var(--app-navigation-padding, 8px) 0 0 calc(2 * var(--app-navigation-padding, 8px) + var(--default-clickable-area));
```

Reply:
But we should also add a variable or component for that to not calculate it all the time in each app...

I'll do it later today.

Reply:
No problem; same code a few lines down.